### PR TITLE
OFI: Add support for FI_MR_RMA_EVENT

### PR DESCRIPTION
**Pull request by jdinan**

### CODE REVIEW

**Comment by jdinan** [comment]
@sayantansur Relying on you to tell me if this actually works.  :)

**Comment by jdinan** [comment]
Appears to have timed out in the first SOS unit test with ```--disable-mr-scalable```...

**Comment by sayantansur** [comment]
I will test today and let you know.

File: src/transport_ofi.c [hunk 1]
```diff
@@ -468,6 +469,11 @@ int allocate_recv_cntr_mr(void)
      * incoming reads/writes and outgoing non-blocking Puts, specifying entire
      * VA range */
 
+#ifdef ENABLE_HARD_POLLING
```
Comment (sayantansur) [inline src/transport_ofi.c]: This should be ifndef ...

**Comment by jdinan** [comment]
Force-pushed an update that adds ```--enable-mr-rma-event``` configury option.

**Comment by jdinan** [comment]
Got the thumbs-up from @sayantansur.  @davidozog What do you think?

**Comment by davidozog** [comment]
Looks good to me.

**Comment by matcabral** [comment]
Friendly comment to get github sending me notifications of this PR, ...  and just to say Hi Guys  :wave:

**Pull request merged**